src/app/screens/DetailArticle/index.tsx

```diff
@@ -29,8 +32,40 @@ function DetailArticle({ route }: Props) {
     if (favoriteCount > favoritesCount) setFavoriteCount(favoriteCount - 1);
     else setFavoriteCount(favoriteCount + 1);
   };
+  const handleDeleteArticle = () => console.log('delete article');
+
+  const handleEditArticle = useCallback(
+    () =>
+      navigation?.navigate(Routes.EditArticle, {
+        article: { ...route?.params?.article, isEditArticle: true }
+      }),
+    [navigation, route]
+  );
+
   return (
     <View style={styles.container}>
+      <View style={styles.containerActions}>
```

Comment:
They can't be visible when user isn't login, can you use?
const currentUser = useSelector((state: State) => state.auth.currentUser);
{currentUser &&( ....)}